doc(release): prepare release notes for modules 23.04 (#2524)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx

## Centreon MAP

### 23.04.4

Release date: `August 03, 2023`

<details>
  <summary>Bug fixes</summary>

> - [Server] Fixed an issue that could cause map edition to become unavailable.

Reviewer: ```suggestion
- [Server] Fixed an issue that could prevent some maps from being edited.
```
